tests/renderMovieScreenings.test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import renderMovieScreenings from "../static/renderMovieScreenings";
+
+describe('renderMovieScreenings', () => {
+
+  beforeEach(() => {
+    document.body.innerHTML = '';
+  });
```

Comment:
There's some sloppy indentation here. At first I thought this and the `test()` invocations weren't on the same level, but then I realized they are on the same level but indented incorrectly.